src/crypto/ecdsa.h

```diff
@@ -10,6 +10,13 @@
 
 namespace crypto
 {
+  // Emulating from mbedtls/ecp.h
+  // Used in raft_types.h
+  // Renato: Can we grow the array at use time?
```

Comment:
What array?

Reply:
I think @rengolin referred to the `std::array<uint8_t, crypto::ECDSA_MAX_LEN> sig;` in `SignedAppendEntriesResponse`. It seems we don't use this struct at all though. Maybe remove it? Perhaps better to turn the array into a vector?

Reply:
If we don't use it, let's remove it!

Reply:
Ah, then removing would be better, yes.